pkg/volume/flexvolume/driver-call.go

```diff
@@ -235,3 +241,29 @@ func handleCmdResponse(cmd string, output []byte) (*DriverStatus, error) {
 
 	return &status, nil
 }
+
+// getDriverCapabilities returns the reported capabilities as returned by driver's init() function
+func getDriverCapabilities(plugin *flexVolumePlugin) (*driverCapabilities, error) {
+	call := plugin.NewDriverCall(initCmd)
```

Comment:
It was a time were the new volumes design was a bit new, so we left the decision about adding the capabilities call for a later time. I agree with a new `capabilities` call to back `isAttachable()` instead of the current call to `init`. It should pretty easy to call `capabilities` then fall back to `init` if the call is not supported.
/cc @chakri-nelluri @saad-ali